app/src/main/java/ch/epfl/sdp/LeaderboardActivity.java

```diff
@@ -0,0 +1,86 @@
+package ch.epfl.sdp;
+
+import android.app.Activity;
+import android.content.Context;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.TextView;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LeaderboardActivity extends AppCompatActivity {
+
+    private List<String> mUsernames = new ArrayList<>();
+    private List<Double> mHealthPoints = new ArrayList<>();
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_leaderboard);
+
+        leaderboardData();
+        initLeaderboardView();
+        setChampions();
+    }
+
+    private void leaderboardData() {
+        mUsernames.add("aaaaa");
```

Comment:
This is not final right ? :)

Reply:
Yeah, just some hardcoded data in order to show something, the true data will come from firebase and will be Player object